src/view/com/modals/LinkWarning.tsx

```diff
@@ -74,7 +74,7 @@ export function Component({text, href}: {text: string; href: string}) {
             onPress={onPressVisit}
             accessibilityLabel={_(msg`Visit Site`)}
             accessibilityHint=""
```

Comment:
Sorry to jump in without invitation, and I know this isn't strictly related to localisation, what about adding the missing accessibility hint here, something like "Opens the linked website"?
```suggestion
            accessibilityHint={_(msg`Opens the linked website`)}
```